services/libs/data-access-layer/src/members/segments.ts

```diff
@@ -1,10 +1,16 @@
+import lodash from 'lodash'
```

Comment:
Don't import entire lodash just use individual npm packages like `lodash.pickby`.

Reply:
Will replace the `lodash` import with specific individual pkgs 👍🏻